Patch 2 (#986)
rocky-linux/documentation: docs/books/admin_guide/01-presentation.md

* **1994**: The commercial distribution Red Hat is created by the company Red Hat, which is today the leading distributor of the GNU/Linux operating system. Red Hat supports the community version Fedora and until recently the free distribution CentOS.

* **1997**: The KDE desktop environment is created. It is based on the Qt component library and the C++ development language.

* **1999**: The GNOME desktop environment is created. It is based on the GTK+ component library.

* **2002**: The Arch distribution is created. Its distinctive is that it offers rolling release (continuous update).

* **2004**: Ubuntu is created by the Canonical company (Mark Shuttleworth). It is based on Debian but includes free and proprietary software.

* **2021**: Birth of Rocky Linux, based on Red Hat distribution.

### Market share

<!--
TODO: graphics with market share for servers and pc.
-->

> Despite its prevalence, Linux remains relatively unknown by the general public, . Linux is hidden in **smartphones**, **televisions**, **internet boxes**, etc. Almost **70% of the websites** in the world are hosted on a Linux or UNIX server!

Reviewer: ```suggestion
Despite its prevalence, Linux remains relatively unknown by the general public. Linux is hidden in **smartphones**, **televisions**, **internet boxes**, etc. Almost **70% of the websites** in the world are hosted on a Linux or UNIX server!
```

There are many graphic environments such as **GNOME**, **KDE**, **LXDE**, **XFCE**, etc. There is something for everyone, and their **ergonomics** hold their own against Microsoft or Apple systems.

So why is there so little enthusiasm for Linux, when this system is practically **virus free**? Could it be because so many editors (Adobe) and manufacturers (Nvidia) do not play the free game and do not provide a version of their software or __drivers__ for GNU/Linux? Perhaps it's fear of change, or the difficulty of finding where to buy a Linux computer, or too few games distributed under Linux. That last excuse at least shouldn't be true for long, with the advent of the game engine Steam for Linux.

![GNOME Desktop](images/01-presentation-gnome.png)

The **GNOME 3** desktop environment no longer uses the concept of desktop but that of GNOME Shell (not to be confused with the command line shell). It serves as a desktop, a dashboard, a notification area and a window selector. The GNOME desktop environment is based on the **GTK+** component library.

![KDE Desktop](images/01-presentation-kde.png)

The **KDE** desktop environment is based on the **Qt** component library.

> It is traditionally recommended for users familiar with a Windows environmnent.

Reviewer: ```suggestion
It is traditionally recommended for users familiar with a Windows environment.
```

**Free** does not mean free!

> **Open Source**: the source codes are available, so it is possible to consult and modify them under certain conditions.

Reviewer: ```suggestion
**Open Source**: the source code is available, so it is possible to consult and modify them under certain conditions.
```

A free software is necessarily Open Source, but the opposite is not true since an Open Source software is separated from the freedom proposed by the GPL license.

#### GPL (General Public License)

> The **GPL** guarantees the author of a software its intellectual property, but allows modification, redistribution or resale of software by third parties, provided that the source codes are included with the software. The GPL is the license that came out of the **GNU** (GNU is Not UNIX) project, which was instrumental in creating Linux.

Reviewer: ```suggestion
The **GPL** guarantees the author of a software its intellectual property, but allows modification, redistribution or resale of software by third parties, provided that the source code is included with the software. The GPL is the license that came out of the **GNU** (GNU is Not UNIX) project, which was instrumental in creating Linux.
```

It offers a real programming language including the classical structures: loops, alternatives, and the common constituents: variables, passing of parameters, and sub-programs. It allows the creation of scripts to automate certain actions (backups, creation of users, system monitoring, etc.).

There are several types of shells available and configurable on a platform or according to the user's preference. A few examples include:

* sh, the POSIX standard shell
* csh, command-oriented shell in C
* bash, Bourne-Again Shell, Linux shell

### Functionalities

* Command execution (checks the command given and executes it)
* Input/Output redirection (returns data to a file instead of writing it on the screen)
* Connection process (manages the user's connection)
* Interpreted programming language (allowing the creation of scripts)

> * Environmental variables (access to information specific to the system during operation)

Reviewer: ```suggestion
* Environment variables (access to information specific to the system during operation)
```
